### ISSUE TITLE
Error on craft 2.5 public

### ISSUE THREAD

**Issue by johanneslamers** [boboldehampsink/auditlog#12]
When saving or editing an entry:

Internal Server Error

Craft\EntryElementType and its behaviors do not have a method or closure named "defineTableAttributes".

**Comment by julzmon** [comment]
Getting same error when I updated a user avatar.

**Comment by Nordicdesign** [comment]
Same issue here, only happens if Audit Log is enabled.

**Comment by lindseydiloreto** [comment]
Also getting this error when updating my user avatar.

Attached a screenshot for reference...

![bug-audit-log](https://cloud.githubusercontent.com/assets/5309692/12637180/1325a0d2-c548-11e5-8d07-76201b1f8b1a.png)

**Comment by boboldehampsink** [comment]
I've published a Craft 2.5 compat update in the develop branch, would you guys mind testing that?

**Comment by lindseydiloreto** [comment]
Just ran a few quick tests with the latest version... seems to be working great so far! :thumbsup:

**Comment by angeldevz** [comment]
I still have the same issue.

**Comment by boboldehampsink** [comment]
Can you add a screenshot of the bug you're seeing?

**Comment by julzmon** [comment]
I am still getting these errors when saving an entry.
![cexception copy](https://cloud.githubusercontent.com/assets/46877/12752688/b39c17c4-c991-11e5-95dc-44af1e3e24ba.png)
![cexception2 copy](https://cloud.githubusercontent.com/assets/46877/12752687/b38b444e-c991-11e5-9152-7a64d62c9159.png)

**Comment by boboldehampsink** [comment]
I see old code in the stacktrace. Did you get the develop branch, as mentioned above?

**Comment by julzmon** [comment]
I thought I did... let me try again.

**Comment by julzmon** [comment]
Yes you're right! Sorry it's working great now thank you.

**Comment by boboldehampsink** [comment]
Just released 0.7.0 which fixes this
